sql/insert.go

```diff
@@ -98,9 +98,11 @@ func (p *planner) Insert(n *parser.Insert) (planNode, error) {
 
 	marshalled := make([]interface{}, len(cols))
 
-	var b client.Batch
+	b := client.Batch{}
+	result := &valuesNode{}
 	for rows.Next() {
 		rowVals := rows.Values()
+		result.rows = append(result.rows, rowVals)
```

Comment:
That would work for now, but not when we get around to implementing RETURNING. how about `result.rows = append(result.rows, append(parser.DTuple(nil), rowVals...))`?